mod.ts

```diff
@@ -85,6 +85,10 @@ export interface BuildOptions {
   redirects?: Redirects;
   /** Package.json output. You may override dependencies and dev dependencies in here. */
   package: PackageJsonObject;
+  /** Package manager used to install dependencies and run npm scripts.
+   * @default npm
+   */
+  packageManager?: string;
```

Comment:
I know that `string` includes this all, but maybe for documentation purposes we could make this type: `"npm" | "yarn" | "pnpm" | string`